src/os/bluestore/BlueStore.cc

```diff
@@ -12584,7 +12584,8 @@ int BlueStore::read(
       l_bluestore_read_onode_meta_lat,
       mono_clock::now() - start1,
       cct->_conf->bluestore_log_op_age,
-      "", l_bluestore_slow_read_onode_meta_count);
+      "", l_bluestore_slow_read_onode_meta_count,
```

Comment:
For distinguishing between IO and scrub for slow ops we only care for actual data transfer functions.
So `read()` `readv()` we want, but `get_onode()`, `_fault_range()` not.

Reply:
Wait, are we really throwing slow event also when metadata retrieval is longer than usual?